### DISCUSSION TITLE
Is it possible to show unreachable URLs in the main flow list?

### DISCUSSION THREAD

**Discussion by Dima-369** [mitmproxy/mitmproxy#7081] [Q&A] [answered]
I test with a URL which does not respond/exist using `httpie` (but same behavior using other approaches to trigger the network request). I have mitmproxy listening on port 9090:

```bash
http GET https://doesnot1321exist.ac --proxy=https://localhost:9090
```

It responds with this:

```
http: error: ProxyError: HTTPSConnectionPool(host='doesnot1321exist.ac', port=443): Max retries exceeded with url: / (Caused by ProxyError('Unable to connect to proxy', OSError('Tunnel connection failed: 502 Bad Gateway'))) while doing a GET request to URL: https://doesnot1321exist.ac/
```

---

In the mitmproxy log, I see this:

```
info: [07:23:01.822][[::1]:50897] client connect
info: [07:23:01.825][[::1]:50897] error establishing server connection: [Errno 8] nodename nor servname
provided, or not known
info: [07:23:01.826][[::1]:50897] client disconnect
```

But nothing pops up in the Flow List in mitmproxy.

Is there any way to also show such failed network requests in the flow view? I use this for testing purposes where URLs can go down like this. Maybe some option, but I did not find anything?

Thanks!

**Comment by Prinzhorn** [comment, marked as answer]
```sh
mitmproxy --set connection_strategy=lazy
```

**Reply by Prinzhorn** [reply]
I've also opened https://github.com/mitmproxy/mitmproxy/issues/7083 because this is not normal
